addons/website_sale_comparison/static/src/interactions/cookies_bar.js

```diff
@@ -0,0 +1,77 @@
+import { CookiesBar } from "@website/interactions/cookies/cookies_bar";
+import { patch } from "@web/core/utils/patch";
+
+patch(CookiesBar.prototype, {
+    /**
+     * @override
+     */
+    setup() {
+        super.setup();
+        // Listen for the custom event to handle the overlap of the compare
+        // button with the cookies bar modal.
+        this.el.addEventListener("cookies_bar_shown", () => {
+            this.handleCompareOverlap();
+        });
```

Comment:
we need to remove this listener as well. Also when we make a custom event, we keep it in uppercase, IDK its mandatory or not, but its easier to differentiate.